src/components/TodoList/TodoList.tsx

```diff
@@ -16,7 +30,76 @@ export const TodoList: React.FC = () => (
     </thead>
 
     <tbody>
-      <tr data-cy="todo" className="">
+      {todos.map(todo => (
+        todo.id === todoId
+          ? (
+            <tr
+              key={todo.id}
+              data-cy="todo"
+              className="has-background-info-light"
+            >
+              <td className="is-vcentered">{ todo.id }</td>
+              <td className="is-vcentered" />
+              <td className="is-vcentered is-expanded">
+                <p className="has-text-danger">
+                  {todo.title}
+                </p>
+              </td>
+              <td className="has-text-right is-vcentered">
+                <button
+                  data-cy="selectButton"
+                  className="button"
+                  type="button"
+                >
+                  <span className="icon">
+                    <i className="far fa-eye-slash" />
+                  </span>
+                </button>
+              </td>
+            </tr>
+          )
+          : (
+            <tr
+              key={todo.id}
+              data-cy="todo"
+              className=""
+            >
+              <td className="is-vcentered">{ todo.id }</td>
+              {!todo.completed && <td className="is-vcentered" />}
+              {todo.completed && (
+                <td className="is-vcentered">
+                  <span className="icon" data-cy="iconCompleted">
+                    <i className="fas fa-check" />
+                  </span>
+                </td>
+              )}
+              <td className="is-vcentered is-expanded">
+                <p className={classNames('has-text-success', {
+                  'has-text-danger': !todo.completed,
+                })}
+                >
+                  {todo.title}
+                </p>
+              </td>
+              <td className="has-text-right is-vcentered">
+                <button
+                  data-cy="selectButton"
+                  className="button"
+                  type="button"
+                  onClick={() => {
+                    onSelectTodo(todo.id);
+                    onSelectUser(todo.userId);
+                  }}
```

Comment:
structure is the same, props differ
i believe you can DRY this code

Reply:
yes absolutely right  , thanks for review!